config-linux.md

```diff
@@ -359,7 +359,12 @@ For more information, see the kernel cgroups documentation about [cpusets][cgrou
 The following parameters can be specified to set up the controller:
 
 * **`shares`** *(uint64, OPTIONAL)* - specifies a relative share of CPU time available to the tasks in a cgroup
-* **`quota`** *(int64, OPTIONAL)* - specifies the total amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+* **`quota`** *(int64, OPTIONAL)* - specifies the replenished amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+    A value of `-1` indicates that the cgroup does not have any bandwidth restriction in place (i.e., an unconstrained bandwidth group).
+    Any (valid) positive value(s) no smaller than `burst` will enact the specified bandwidth limit.
+* **`burst`** *(uint64, OPTIONAL)* - specifies the maximum amount of accumulated time in microseconds for which all tasks in a cgroup can run additionally for burst during one period (as defined by **`period`** below)
+    A value of `0` for indicates that the cgroup can not accumulate any unused bandwidth. It makes the traditional bandwidth control behavior unchanged.
+    Any (valid) positive value(s) no larger than `quota` will enact the cap on unused bandwidth accumulation.
```

Comment:
I think this would be more consistent if we assume that not specifying this optional field is the way to get "no limit", and I think it would also make sense to be more clear about values bigger than `quota` are not just ignored but rather an error, perhaps something like this?

```suggestion
    If specified, this value MUST be no larger than `quota` (runtimes MAY generate an error).
```

**Edit:** coming back to this after https://github.com/opencontainers/runc/pull/3205#discussion_r705484770, I'm guessing that's related? (And likely why the "negative values" bits were added to `quota`?)